pkg/minikube/mustload/mustload.go

```diff
@@ -77,10 +81,49 @@ func Partial(name string, miniHome ...string) (libmachine.API, *config.ClusterCo
 		}
 		exit.Error(reason.HostConfigLoad, "Error getting cluster config", err)
 	}
+	if cc.Driver == driver.HyperV {
```

Comment:
I feel like this is better to be in the cmds that actually use tunnel, 
how about moving this logic to cmd/tunnel.go we dont wanna exit on Loading the machine, for things like "status"